stats.py

```diff
@@ -390,7 +414,7 @@ def load_dataframe():
     return pd.DataFrame.from_dict(parse_solutions(walk_problems()), "index")
 
 
-def solutions_paths(df):
+def solutions_paths(df, from_files=None):
```

Comment:
im not sure about implementing this here, maybe I am repeating some code out side ?

Reply:
well, this is a little hacky, but since this change little code we can accept that for now

Reply:
but this really should not be here.... probably we'll need a full refactor of `stats.py` later, probably neither I understand all this script although I wrote all this shit.. sorry for you have to working with this crazy build system hahaha

Reply:
hahaha x), its routine don't worry :p

Reply:
> probably we'll need a full refactor of stats.py later

I agree on that, some changes and classes maybe make things clearer and easier de fix

Reply:
Using functions is ok, the main problem is rely on `pandas` to do a lot of operations. But probably I will not remove that because I need pandas these days to generate some statistics stuff (like graphs) which this was the main reason to write this script (not only for testing/building), so doesn't makes sense remove that. But indeed I need write in a more sane way.